lib/generate_rota.rb

```diff
@@ -82,7 +83,8 @@ def to_h
   def to_csv
     slots = slots_filled(@people)
     weeks = slots.map { |slot| slot[:week] }.uniq.max
-    roles = slots.map { |slot| slot[:role] }.uniq
+    # roles = slots.map { |slot| slot[:role] }.uniq
```

Comment:
We should remove this commented out code.

Reply:
Agreed. I'll make a PR with this comment removed and a comment added about the seed.